fbssdc/.travis.yml

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+language: python
```

Comment:
I suppose this file will have to be modified somehow or manually called from top-level .travis.yml?

Reply:
Yes I will figure out how to get the Travis files to play together

Reply:
It would be nice to get this done before the merge.